1-Draft/RFC00XX-Update-Error-View.md

```diff
@@ -0,0 +1,182 @@
+provides a level of detail that obscures the inner exception message from being read by
+new or occasional PowerShell users.
+To improve both comprehension and the troubleshooting experience, a single string of text
+containing the error will be displayed.  A cmdlet will provide the full error details when desired.
+
+## Motivation
+
+The on-screen experience, when receiving an error message,
+is controlled through the views NormalView (the default) and CategoryView. These are user selectable though the preference variable $ErrorView.
+The addition of a “ConciseView” as default that contains only specific error
+information producing a single line on screen and in logs will improve customer success.
+
+Comprehensive error information is available to the customer through
+accessing the error object. For new and occasional customers,
+a cmdlet “Resolve-ErrorRecord” will simplify the process of getting detailed error information, improving customers success.
+
+## Specification
+
+The proposal is to add a new single-line default error message view called ConciseView
+and add a new cmdlet (Resolve-ErrorRecord) to provide detailed error information.
+
+__Key Design Considerations__
+
+1. To reduce confusion and improve debugging success,
+error messages by default should produce a single line of text, including the word “ERROR:” to make consistent with Verbose and Warning messages. This is added to $errorView as view named "ConciseView".
+
+- $ErrorView should contain these views
+
+    + NormalView
+    + CategoryView
+    + ConciseView
+
+- Error Message syntax for ConciseView
+
+```powershell
+ERROR: <Exception Message>: <Positional Info>
+ERROR: Cannot find path ‘C:\blah’ because it does not exist: At line:1 char:1
+```
+
+2. A new cmdlet “Resolve-ErrorRecord” will produce detailed error information similar to Format-List * -Force.
+
+- Resolve-ErrorRecord will provide the following:
+
+    + Display Last Error ($Error[0]) – default behavior
+    + Accept Pipeline input – support $error[1] |
+    + Option for the first X errors in the session
+    + Option for the Last X errors in the session
+
+- Resolve-ErrorRecord syntax
+
+```powershell
+Resolve-ErrorRecord  [-InputObject <psobject>] [-Newest <Int32>] [-First <Int32>] [-All] [<CommonParameters>]
+```
+
+First parameter set
+
+- Newest
+
+    + Datatype: int32
+    + specifies one or more of last errors to display
+    + Not required
+
+- First
+
+    + Datatype: Int32
+    + Specifies one or more of the first errors that occurred
+    + Not required
+
+- All
+
+    + Datatype: Switch
+    + Specifies display all errors in session
+    + Not required
+
+Examples
+
+```powershell
+# Displays details of the last error displayed
+Resolve-ErrorRecord
+
+# Displays error details from pipeline error object
+$error[1] | Resolve-ErrorRecord
+
+# Display detailed error information for the last 5 errors
+Resolve-ErrorRecord -Last 5
```

Comment:
Looks like you originally had `-Last` but changed to `-Newest` without updating the examples

Reply:
The examples still show `-First` and `-Last`, but the cmdlet definition on line 71 only shows `-Newest`.  I think keeping just `-Newest` makes sense so the examples need to be updated.